webapp/src/app.tsx

```diff
@@ -2787,13 +2787,18 @@ export class ProjectView
         })
     }
 
-    newProject(options: ProjectCreationOptions = {}) {
+    newProject(options: ProjectCreationOptions = {}, isFirstProject?: boolean) {
```

Comment:
we don't need to add a new parameter for this one, just add it into `ProjectCreationOptions` here https://github.com/microsoft/pxt/blob/master/pxteditor/editor.ts#L121 and pass in the blob (e.g. how `name` and `languageRestriction` are passed in down in projects.tsx)